packages/react/src/create-strict-api/index.ts

```diff
@@ -36,7 +20,9 @@ interface CompiledAPI<TSchema extends CompiledSchema> {
    * <div css={redText} />
    * ```
    */
-  css(styles: CSSStyles<TSchema>): StrictCSSProperties;
+  css<TStyles extends ApplySchema<TStyles, TSchema>>(
```

Comment:
To get the types working I built up `css` and then extrapolated it into `cssMap`. I should probably change the return type so it doesn't work with `xcss` prop...

Reply:
Yeah. Will make a task!